### DISCUSSION TITLE
Cannot generate html-report

### DISCUSSION THREAD

**Discussion by jackshavryhin** [cucumber/common#2177] [General]
I am running my test, and after i install `npm install -g cucumber-html-reporter`, but when i try run this command `cucumber-html-reporter -i results.json -o report.html
` every time getting error "no such file or directory" even if i using full path to cucumber-html-reporter package.
Maybe someone know, how i can solve this issue ?

My code example:

`func TestPT_01(t *testing.T) {

	file, err := os.OpenFile("results.cucumber", os.O_CREATE|os.O_WRONLY, 0644)
	if err != nil {
		t.Fatalf("Failed to open results.json file: %v", err)
	}
	defer file.Close()

	suite := godog.TestSuite{
		ScenarioInitializer: InitializeScenario,
		Options: &godog.Options{
			Format:        "cucumber",
			Paths:         []string{"pt_01.feature"},
			Output:        file, // Output file for the JSON report
			Strict:        false,
			StopOnFailure: false,
			TestingT:      t,
		},
	}

	if suite.Run() != 0 {
		t.Fatal("non-zero status returned, failed to run feature tests")
	}

	os.Setenv("NODE_PATH", "/usr/local/lib/node_modules")

	cmd := exec.Command("/usr/local/lib/node_modules/cucumber-html-reporter", "-i", "results.cucumber", "-o", "results.html")
	err = cmd.Run()
	if err != nil {
		t.Fatalf("Failed to generate HTML report: %v", err)
	}
}`

**Comment by mpkorstanje** [comment]
The `cucumber-html-reporter` is written by gkushang. You might want to ask over there. https://github.com/gkushang/cucumber-html-reporter

**Comment by kush27082000** [comment]
@jackshavryhin Did you got the answer of your query regarding how to generate reports?  If yes then can you please share the solution along with demo code for generation of report.
